### ISSUE TITLE
debian packaging

### ISSUE THREAD

**Issue by fossfreedom** [TheBigW/DRC#1]
some initial issues:

make file appears to be wrong - suggestion for the contents?

<pre>
DESTDIR=
SUBDIR=/usr/lib/rhythmbox/plugins/DRC/
DATADIR=/usr/share/rhythmbox/plugins/DRC/

all:
clean:
    - rm *.pyc

install:
    install -d $(DESTDIR)$(SUBDIR)
    install -m 644 *.py $(DESTDIR)$(SUBDIR)
    install -m 644 LICENSE.txt $(DESTDIR)$(SUBDIR)
    install -m 644 DRC.plugin* $(DESTDIR)$(SUBDIR)  
    install -d $(DESTDIR)$(DATADIR)
    install -m 644 calcFilter* $(DESTDIR)$(DATADIR)
    install -m 644 measure* $(DESTDIR)$(DATADIR)
    install -m 644 erb44100.drc $(DESTDIR)$(DATADIR)
    install -m 644 *.glade $(DESTDIR)$(DATADIR)
    
</pre>

README:

  the README instructions for the ubuntu install lists the extra packages required - but python3-autopilot and alsa-utils as suggested in install_deps.sh is not listed.

python3-autopilot:

adding the above as a dependency pulls in 18Mb of other packages - are you really sure this is a required package dependency?

**Comment by fossfreedom** [comment]
debian package throws warnings about calcFilter and measure being potential script files but no executable flag is set - the Makefile is resetting the permissions to be 644 - what should the permission number be?

**Comment by TheBigW** [comment]
you made a point here: python-autopilot was just the easiest way to get python-gconf, which I used for the configuration. I got rid of it now and switched to XML config (which I should have done quite some time ago.... ). So python3-autopilot is not needed any more.
I also adapted the makefile. I guess for the scripts the permission shall be 755 (if I googled that right...). Also fixed the data dir install part.
At least I got the installation working so far that it works now for me.

Thanks a lot for those hints!
